javaslang/src-gen/main/java/javaslang/Tuple2.java

```diff
@@ -238,4 +238,8 @@ public String toString() {
         return "(" + _1 + ", " + _2 + ")";
     }
 
+    public static <T1, T2> Tuple2<Seq<T1>, Seq<T2>> sequence(Seq<Tuple2<T1, T2>> tuples) {
```

Comment:
Great idea!

Please let's tweak the parameter types a bit:

```java
public static <T1, T2> Tuple2<Seq<T1>, Seq<T2>> sequence(Iterable<Tuple2<? extends T1, ? extends T2>> tuples)
```

_(Note: I did not suggest `Iterable<? extends Tuple...` because Tuples are final.)_